libres/tests/analysis/std_enkf.cpp

```diff
@@ -0,0 +1,39 @@
+#include <catch2/catch.hpp>
+
+#include <ert/analysis/std_enkf.hpp>
+
+TEST_CASE("std_enkf", "[analysis]") {
```

Comment:
I think this is a nice candidate to use the GIVEN - WHEN - THEN pattern in `catch2` as shown here:

https://github.com/catchorg/Catch2/blob/v2.x/docs/tutorial.md#bdd-style

It's a bit of a pain to get used to at first, but I think it's quite nice.

Reply:
> I think this is a nice candidate to use the GIVEN - WHEN - THEN pattern in catch2 as shown here:

OK - I'll take the pill. Thank you for the pointer.